### DISCUSSION TITLE
Is anyone using the XRD validation webhook?

### DISCUSSION THREAD

**Discussion by negz** [crossplane/crossplane#3749] [Q&A]
Per https://github.com/crossplane/crossplane/issues/3748 we're considering turning these on by default. Has anyone tried them out by running Crossplane with `--set webhooks.enabled=true`? How are they working?

**Comment by blakebarnett** [comment]
Had no idea they existed, but I would definitely like the functionality! Would this mean support for CRD validation rules in an XRD?

**Comment by negz** [comment]
@blakebarnett No, this validates that the actual `CompositeResourceDefinition` is valid, not the XR it defines. That said we are starting to think about better validating XRs which is what bought this to mind for me.

**Reply by blakebarnett** [reply]
> @blakebarnett No, this validates that the actual `CompositeResourceDefinition` is valid, not the XR it defines. That said we are starting to think about better validating XRs which is what bought this to mind for me.

Got it, I'll turn it on and see how it works.
